pranav/cart/spec.rb

```diff
@@ -0,0 +1,123 @@
+require "./main.rb"
+
+describe Cart do
+  describe "#cal_gst"
+    context "calculate GST on gross_amount" do      
+      before do
+        s_gst = 8
+        c_gst = 8
+      end      
+      it "return GST amount on gross_amount" 
+        true_amount = (gross_amount / 100) * (c_gst + s_gst)
```

Comment:
in test cases, we should not use dynamic values.
we have to use static values to compare against actual values.

Reply:
Fixed.